### ISSUE TITLE
Error while importing portable project

### ISSUE THREAD

**Issue by grossmj** [GNS3/gns3-gui#2219]
https://gns3.com/qa/error-importing-portable-project

```
ImportProjectWorker thread stopping with an error: Could not import the project: [Errno 13] Permission denied: 'None'
```

**Comment by noplay** [comment]
I already fixed it in 2.1 https://github.com/GNS3/gns3-gui/issues/2082

**Comment by MeeListen** [comment]
Hello, 
I am having this issue, a colleague sent me a project that i'm trying to upload into my gns3 setup but i get this error while importing. i am running GNS3 2.0.3 and  i am so afraid to upgrade because i have a lot of projects saved that i dont want to loose. Please kindly help
